lbu_files/nemeth.ctb

```diff
@@ -22,8 +23,56 @@
 #  License along with liblouis. If not, see
 #  <http://www.gnu.org/licenses/>.
 
-include en-chardefs.cti All character definition opcodes
+space \x001b 1b escape character for html back-translation
+include loweredDigits6Dots.uti
+include latinLetterDef6Dots.uti
+include latinUppercaseComp6.uti
+replace \x00AF \x2015
+punctuation ( 12356
+punctuation } 12456
+punctuation ] 124567
+punctuation ? 1456
+punctuation : 156
+punctuation ) 23456
+punctuation ! 2346
+punctuation { 246
+punctuation [ 2467
+punctuation ' 3
+punctuation - 36
+punctuation . 46
+punctuation " 5
+punctuation ; 56
+punctuation , 6
+sign & 12346
+sign $ 1246
+sign | 1256
+sign \\ 12567
+sign % 146
+sign * 16
+sign # 3456
+punctuation ` 4
+sign _ 456
+sign ^ 457
+sign @ 47
+math = 123456
+math < 126
+math / 34
+math > 345
+math + 346
+math ~ 45
+punctuation : 456-25-0 #Added in 2023
```

Comment:
Please don't put stuff like `#Added in 2023` in. I can tell that from the commit history.